packages/datagrid/src/containers/datagrid.container.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+import Component from '../components/';
+
+export default function DataGrid(props) {
+	const onFocusedCell = event => {
+		if (props.onFocusedCell) {
+			props.onFocusedCell();
+		}
+
+		if (props.actionCreators && props.actionCreators.onFocusedCell) {
```

Comment:
BTW i just saw that actionsCreators is an object. Don't you prefer to flatten it to avoid this multiple check on actionCreators and the specific actionsCreator